core/orchestrator.go

```diff
@@ -36,7 +36,7 @@ import (
 
 const maxSegmentChannels = 4
 
-var transcodeLoopTimeout = 1 * time.Minute
+var transcodeLoopTimeout = 70 * time.Second
```

Comment:
Is the idea here that we were timing out at the same time as the broadcaster does and so timed out before we had a chance for the B to end the session?

Reply:
That's it yep

Reply:
Maybe add a comment that this timeout is set to be greater than B timeout?

Reply:
Maybe add a comment that this timeout is set to be greater than B timeout? I'm not sure if we want to have it as tight as possible, +10 sec probably should work fine.